tools/generator/test/CreateXCSchemesTests.swift

```diff
@@ -0,0 +1,184 @@
+import PathKit
+import XcodeProj
+import XCTest
+
+@testable import generator
+
+class CreateXCSchemesTests: XCTestCase {
+    let filePathResolver = FilePathResolver(
+        internalDirectoryName: "rules_xcodeproj",
+        workspaceOutputPath: "examples/foo/Foo.xcodeproj"
+    )
+    let pbxTargetsDict: [TargetID: PBXTarget] =
+        Fixtures.pbxTargetsWithDependencies(
+            in: Fixtures.pbxProj(),
+            targets: Fixtures.targets
+        )
+
+    func assertScheme(
+        schemesDict: [String: XCScheme],
+        targetName: String,
+        shouldExpectBuildActionEntries: Bool,
+        shouldExpectTestables: Bool,
+        shouldExpectBuildableProductRunnable: Bool
```

Comment:
```suggestion
    func assertScheme(
        schemesDict: [String: XCScheme],
        targetName: String,
        shouldExpectBuildActionEntries: Bool,
        shouldExpectTestables: Bool,
        shouldExpectBuildableProductRunnable: Bool,
        file: StaticString = #filePath,
        line: UInt = #line
```

And then can you pass `file` and `line` to the `XCT` functions to get the correct line to show up when there are failures?

Reply:
Done.